admin-dev/themes/new-theme/js/pages/product/edit/customizations-manager.ts

```diff
@@ -60,9 +71,9 @@ export default class CustomizationsManager {
     this.eventEmitter.emit(ProductEventMap.customizations.rowAdded, {index});
   }
 
-  removeCustomizationField(event) {
-    const $deleteButton = $(event.currentTarget);
-    const modal = new ConfirmModal(
+  removeCustomizationField(event: JQuery.ClickEvent<HTMLElement, undefined, any, any>): void {
```

Comment:
```suggestion
  removeCustomizationField(event: JQuery.ClickEvent): void {
```
This should be enough no?